tasks/coverage/parser_typescript.snap

```diff
@@ -3880,6 +3880,39 @@ Expect to Parse: "compiler/withStatementInternalComments.ts"
    ·       ────
    ╰────
 
+Expect to Parse: "conformance/classes/propertyMemberDeclarations/autoAccessor2.ts"
```

Comment:
Can you double check this?

Reply:
It seems like we need to register for auto accessor? https://github.com/microsoft/TypeScript/blob/main/tests/cases/conformance/classes/propertyMemberDeclarations/autoAccessor2.ts